sled-agent/src/params.rs

```diff
@@ -623,10 +595,6 @@ impl TryFrom<ServiceZoneService>
 }
 
 /// Used to request that the Sled initialize multiple services.
-///
-/// This may be used to record that certain sleds are responsible for
-/// launching services which may not be associated with a dataset, such
-/// as Nexus.
```

Comment:
This comment has been removed, as now these requests handle both kinds of services (with and without datasets).